stats/bin/spark-stats.js

```diff
@@ -1,23 +1,26 @@
 import '../lib/instrument.js'
 import { createApp } from '../lib/app.js'
-import { getPgPools } from '@filecoin-station/spark-stats-db'
 
 const {
   PORT = '8080',
   HOST = '127.0.0.1',
   SPARK_API_BASE_URL = 'https://api.filspark.com/',
-  REQUEST_LOGGING = 'true'
+  REQUEST_LOGGING = 'true',
+  DATABASE_URL,
+  EVALUATE_DB_URL
```

Comment:
Let's add default values for both `env` variables:

```suggestion
  DATABASE_URL = 'postgres://localhost:5432/spark_stats',
  EVALUATE_DB_URL = 'postgres://localhost:5432/spark_evaluate'
```